src/components/Root/index.tsx

```diff
@@ -37,6 +43,11 @@ const RootConsumer = (): React.ReactElement | null => {
       } catch (err) {
         logError(Errors._904, err.message)
         setIsError(true)
+      } finally {
+        // If chain is not returned from CGW, revert to default
+        if (!isValidChainId(chainId)) {
+          setChainId(DEFAULT_CHAIN_ID)
```

Comment:
Maybe use `_getChainId()` to avoid extra dependencies and make a redirect to `/app/{chainName}` here?